[2016-05-09 12:03:47,553] {models.py:126} INFO - Filling up the DagBag from /Users/bruno/Development/Intelivix/etl-example/dags/update_progress.py
[2016-05-09 12:03:47,553] {models.py:204} INFO - Importing /Users/bruno/Development/Intelivix/etl-example/dags/update_progress.py
[2016-05-09 12:03:47,837] {models.py:296} INFO - Loaded DAG <DAG: update_progress>
[2016-05-09 12:03:56,701] {models.py:126} INFO - Filling up the DagBag from /Users/bruno/Development/Intelivix/etl-example/dags/update_progress.py
[2016-05-09 12:03:56,701] {models.py:204} INFO - Importing /Users/bruno/Development/Intelivix/etl-example/dags/update_progress.py
[2016-05-09 12:03:56,975] {models.py:296} INFO - Loaded DAG <DAG: update_progress>
[2016-05-09 12:03:57,245] {models.py:1001} INFO - 
--------------------------------------------------------------------------------
Attempt 1 out of 2
--------------------------------------------------------------------------------

[2016-05-09 12:03:57,464] {models.py:1041} INFO - Executing <Task(PythonOperator): extract_juntada_task> on 2016-04-30 00:00:00
[2016-05-09 12:03:57,671] {python_operator.py:66} INFO - Done. Returned value was: 0
